src/pages/Human_Practices.pug

```diff
@@ -134,6 +134,40 @@ block article
         
     - var imgpath = "" + require("../assets/img/Human Practices/2021-06-11_Meeting-Rother.png")["default"]
     +image(4, imgpath, "Description of the image")
+    
+     :markdown-it(html)
+     
+        # Meeting with Dr. Martin Zimmermann
+        
+        Especially at the beginning of our project, we had severe problems with our analytics. Gels did not run well, samples were not visible and many members of our team had little to no lab experience.
+        Dr. Martin Zimmermann is not only the study coordinator of the Biological Sciences faculty, but he is also an expert in many biological fields including biological analysis methods.
+        During the whole project, we repeatedly exchanged E-Mails, where he answered our questions with much detail and knowledge.
+        We met with Dr. Zimmermann at the end of Mai. He introduced denaturated Polyacrylamide gels to us and helped us establish the right analytics for our samples. Among other things, we discussed gel runtime, gel size, the right voltage for our samples, and staining methods of our gels.
+        To get a better resolution Dr. Zimmermann advised us to use TBE buffer, which has a higher buffer capacity.
+        Furthermore, he suggested silver staining as a sustainable method to stain PAA gels and longer primers to establish better reaction conditions for our Enzym, which we implemented rapidly.
+        At the end of our Meeting, Dr. Zimmermann once more drew attention to Polyacrylamide. He advised us to be extra careful when handling the neurotoxic chemical. Click here to see our safety guidelines.
+
+    :markdown-it(html)
+    
+        # Meeting with Dr. Sebastian Palluk
+    
+        Dr. Palluk is co-founder and CTO of Ansa Biotechnologies, a startup developing enzymatic DNA synthesis technologies. When we were younger he participated in the iGEM competition himself and found it challenging that synthesizing DNA was such a difficult task. Nowadays he works with a genetically modified TdT for biological Applications and aims to improve the synthesis system for future implications.
+        Mr. Palluk was probably one of our most important experts. We met with him three times in total and had intensive E-Mail contact during the whole project.
+        In our first meeting, we started off presenting our ideas and two approaches to encode data on DNA.
+        Furthermore, we discussed four different ideas to prevent the secondary structure formation of our ssDNA primers and the affinity of the TdT for 3´OH base modifications.
```

Comment:
```suggestion
        Furthermore, we discussed four different ideas to prevent the formation of secondary structure of ssDNA and the affinity of the TdT for 3´OH base modifications.
```